CalculatorTests/CalculatorTests/Tests/CalculatorPageTests.cs

```diff
@@ -18,9 +18,34 @@ public void SetUp()
         {
             var options = new ChromeOptions { AcceptInsecureCertificates = true };
             driver = new ChromeDriver(options);
+            SetSettingsToDefault();
             driver.Url = "https://localhost:5001/Calculator";
         }
 
+
+        public void SetSettingsToDefault()
+        {
+            driver.Url = "https://localhost:5001/Settings";
+
+            IWebElement dateFormatField = driver.FindElement(By.Id("dateFormat"));
+            IWebElement numberFormatField = driver.FindElement(By.Id("numberFormat"));
+            IWebElement defaultCurrencyField = driver.FindElement(By.Id("currency"));
+            IWebElement saveButton = driver.FindElement(By.Id("save"));
+            SelectElement dateFormatDropdown = new SelectElement(dateFormatField);
+            SelectElement numberFormatDropdown = new SelectElement(numberFormatField);
+            SelectElement defaultCurrencyDropdown = new SelectElement(defaultCurrencyField);
+
+            new WebDriverWait(driver, TimeSpan.FromSeconds(200))
```

Comment:
200 seconds is too much.
```suggestion
            new WebDriverWait(driver, TimeSpan.FromSeconds(10))
```

Reply:
Fixed